addon/components/new-osf-navbar/component.js

```diff
@@ -34,6 +34,9 @@ export default Ember.Component.extend(hostAppName, AnalyticsMixin, {
         if (appName === 'Dummy App') {
             appName = 'Home';
         }
+        if (appName === 'Ember OSF Web') {
```

Comment:
And this needs to be redone completely, yes? We have our own navbar, and this wouldn't be true in ember-osf anymore, I'm guessing.